.env.example

```diff
@@ -3,4 +3,8 @@ NEXT_PUBLIC_GISCUS_REPOSITORY_ID=
 NEXT_PUBLIC_GISCUS_CATEGORY=
 NEXT_PUBLIC_GISCUS_CATEGORY_ID=
 NEXT_PUBLIC_UTTERANCES_REPO=
-NEXT_PUBLIC_DISQUS_SHORTNAME=
\ No newline at end of file
+NEXT_PUBLIC_DISQUS_SHORTNAME=
+NEXT_PUBLIC_MAILCHIMP_API_KEY=
+NEXT_PUBLIC_MAILCHIMP_API_SERVER=
+NEXT_PUBLIC_MAILCHIMP_AUDIENCE_ID=
+NEXT_PUBLIC_BUTTONDOWN=
```

Comment:
The variables only used in API routes don't have to be prefixed with NEXT_PUBLIC. This way they're exposed to the browser too. https://nextjs.org/docs/basic-features/environment-variables